Kickstarter-iOS/Features/MessageBanner/Controller/MessageBannerViewController.swift

```diff
@@ -9,14 +9,22 @@ public protocol MessageBannerViewControllerPresenting {
     -> MessageBannerViewController?
 }
 
+public protocol MessageBannerViewControllerDelegate: AnyObject {
+  func messageBannerViewDidHide(type: MessageBannerType)
```

Comment:
Nit: it is more canonical to pass the calling object itself back in the delegate method. So the method would look like `messageBannerViewDidHide(view: MessageBannerViewController)` and then `bannerType` could be a `public private(set)`

Reply:
this method only needs to be aware of the banner type though. what value do we gain from passing the entire object?

Reply:
Ha, we gain nothing right now, but if you _did_ need the dismiss behavior to do something else in the future, it would be simpler to implement. However, that's why I'm thinking of this as a nit; it's really more of a style detail.

You may not need the delegate method at all if you're able to pull the `userInteractionDisabled` stuff into the view controller.